src/errors.ts

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2015-2022 The Matrix.org Foundation C.I.C.
```

Comment:
We can't guarantee that it was 2015 - 2022 because the previous file didn't have any headers. If you're assigning your copyright to the foundation, it'd just be 2022.